content/plugins/commons-chunk-plugin.md

```diff
@@ -182,6 +187,23 @@ new webpack.optimize.CommonsChunkPlugin({
 })
 ```
 
+To add further control to an implicit common vendor chunk, you can additional logic before the `node_modules` context check to prevent
+specific modules from being included in the vendor chunk (such as stylesheets you want to keep in the original chunk):
```

Comment:
Reading through this example leaves the question open as why this would be useful. I mean from the nice documentation of the module object from above this should be easy to achieve. But I do not see the importance in general to keep the CSS files in a different chunk. If there's a good use case, please elaborate. Otherwise, I'd drop this example.